cudax/include/cuda/experimental/__memory_resource/any_resource.h

```diff
@@ -95,10 +105,12 @@ class basic_any_resource
     }
   }
 
-#ifndef DOXYGEN_SHOULD_SKIP_THIS // Doxygen misparses the below constructor, so dumb it down for it.
+  //! @brief Conversion from a \c basic_any_resource with the same set of properties but in a different order.
+  //! This constructor also handles conversion from \c async_any_resource to \c any_resource
+  //! @param __ref The other \c basic_any_resource.
```

Comment:
Important: There is no parameter `__ref`, only `__other`.
```suggestion
  //! @param __other The other \c basic_any_resource.
```